packages/components/src/local/mapping/helpers/createGrid.ts

```diff
@@ -50,17 +51,55 @@ const createGrid = (bounds: L.LatLngBounds, tileDiameterMins: number): SergeGrid
     const newCell: SergeHex<{}> = cell as SergeHex<{}>
     newCell.name = cellName(newCell)
 
-    // generate the cenll centre in Leaflet coords
+    // generate the cell centre in Leaflet coords
     const centreHex: Point = cell.toPoint()
     newCell.centreLatLng = toWorld(centreHex, correctedOrigin, tileSizeDegs / 2)
     return newCell
   })
   const unTyped: any = newCells
   const asSerge: Grid<SergeHex<{}>> = unTyped as Grid<SergeHex<{}>>
 
+  // sort out corner offset
+  // the hexes all have the same corners object, so just use the first one
+  const hexOne = asSerge[0]
+
+  // get the coordinates of the centre of the hex, relative
+  // to the top-left origin
+  const centreH = hexOne.center()
+
+  // and the coords of the top-left origin
+  const cellOrigin = hexOne.coordinates()
+
+  // capture the offset between a cell centre, and the cell origin
+  const centreOffset = L.point(centreH).subtract(L.point(cellOrigin))
+  
   const sergeGrid: SergeGrid<SergeHex<{}>> = asSerge as SergeGrid<SergeHex<{}>>
   sergeGrid.origin = correctedOrigin
   sergeGrid.tileDiameterDegs = tileSizeDegs
+  sergeGrid.centerOffset = centreOffset
+  sergeGrid.toScreen = (point: L.LatLng): PointLike => {
+    return toScreen(point, sergeGrid.origin, sergeGrid.tileDiameterDegs / 2)
+  }
+  sergeGrid.toWorld = (point: PointLike): L.LatLng => {
+    return toWorld(point, sergeGrid.origin, sergeGrid.tileDiameterDegs)
+  }
+  sergeGrid.toWorld2 = (point: PointLike, origin: L.LatLng, delta: number): L.LatLng => {
```

Comment:
Can you find a better name for this? What is the difference between `toWorld` and `toWorld2`?

Reply:
The function comment is in the class definition.

I've renamed it to `toWorldLocal`

Reply:
Correction, I've now deleted it. It wasn't used.